### DISCUSSION TITLE
国服在选择英雄时不跳出配置符文的框框

### DISCUSSION THREAD

**Discussion by xxbash** [cangzhang/ChampR#267] [General]
感谢大佬开发这个软件，但是好像有段时间不能用了，大佬是放弃国服了吗，一直用的v2.0.1-fix版本，游戏及软件安装路径中没有中文，但是选择英雄的时候弹不出来改符文的界面
游戏客户端版本：V13.11
[![pCVEfOS.md.png](https://s1.ax1x.com/2023/06/10/pCVEfOS.md.png)](https://imgse.com/i/pCVEfOS)

**Comment by cangzhang** [comment]
可以试试新版本 https://github.com/cangzhang/champ-r/releases/tag/v2.0.2-b1
